docs/components/base/_index.md

```diff
@@ -11,7 +11,7 @@ imageAlt: "base"
 # SMEs: Steve B
 ---
 
-A *base* is the platform that the other parts of a mobile robot attach to.
+A {{< glossary_tooltip term_id="base" >}} is the platform that the other parts of a mobile robot attach to.
```

Comment:
Link is broken because of staging. I can't use relative links on these because I don't know where they'll be used... I'll look into what I can do. 

I'll remove the link here though